orm/__init__.py

```diff
@@ -3,6 +3,8 @@
 from gene import Gene
 from transcript import Transcript
 from individual import Individual
+from patient import Patient
```

Comment:
@pontikos should this line be here? It's making travis cry... 
I might have introduced it during a merge conflict...
Or are there any other commits I should pull over from the other dev...